### DISCUSSION TITLE
Systemd Unit File Setting User

### DISCUSSION THREAD

**Discussion by ggalt** [toverainc/willow-inference-server#124] [General]
Anyone have thoughts on what user should be used for running the WIS via systemd?  My current unit file looks like this:
```

[Unit]
Description=Willow Inference Server
After=docker.service
Requires=docker.service

[Service]
User=[my_user]
Group=[my_user_group]
ExecStart=/bin/bash /home/[my_user]/willow-inference-server/utils.sh run

[Install]
WantedBy=multi-user.target


```

Would using the "nobody" user be better?  What would be the issue with using:
```
[Service]
User=nobody
Group=docker
```

Does anyone have a better idea or thoughts on this?

**Comment by nikito** [comment]
Is there a reason you are trying to use systemd? Docker usually already runs as a service on a machine so there isn't a need to use systemd, unless your docker install was somehow set to not have the service automatically start at boot. If you are trying to have WIS run automatically in docker when it comes up, you can add the -d option to your utils.sh command, like this:
./utils.sh run -d
Or you can edit the restart policy of the docker container to unless-stopped.

**Reply by ggalt** [reply]
I have to admit I'm not that familiar with Docker, so I was unaware of the option.  I'll give that a try.
